### PR TITLE
drop to upload file is done

### PR DESCRIPTION
**Pull request by Sina-pz**

### CODE REVIEW

**Review by dhaval2989 — COMMENTED** [review]
Create a custom class instead of overridden the global class in base.scss

File: src/app/reusable/upload/upload.component.html [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<div class="container" appNvFileUpload (fileDropped)="onFileDropped($event)">
+  <div class="d-flex flex-column  align-items-center">
+    <input type="file" #fileDropRef id="fileDropRef" multiple (change)="fileBrowseHandler($event)" />
+    <svg class="align-self-center mt-5" id="ico.laboperations" xmlns="http://www.w3.org/2000/svg" width="36" height="32" viewBox="0 0 36 32">
```
Comment (veeranjaneyuludavuluri) [inline src/app/reusable/upload/upload.component.html:4]: dont' use svg code directly, please export that svg image and place it in assets/icons folder and reuse them

**Review by Sina-pz — COMMENTED** [review]
> Create a custom class instead of overridden the global class in base.scss

Sure

**Review by veeranjaneyuludavuluri — APPROVED** [review]
Good to go

**Review by dhaval2989 — APPROVED** [review]

**Pull request merged**